app/controllers/concerns/contributable.rb

```diff
@@ -78,6 +78,10 @@ def new_contribution
   end
 
   def formify_contribution(contribution)
+    contribution.ore_aggregation.edm_web_resources.each do |wr|
```

Comment:
I don't think this belongs inside formify_contribution, it's a distinct action which is also not needed in the new or edit methods. Could it be in it's own method and only be called before formify_contributions in the controller methods where needed?